File: Python/Objetos.py
#Python es un lenguaje orientado a objeto
# es un paradigma que ha ganado una gran popularidad en los últimos años debido a su capacidad para crear aplicaciones,
# más robustas, flexibles y fáciles de mantener. Esta metodología de desarrollo se basa en la idea de que los programas,
#se pueden organizar como una colección de objetos interconectados.
# Definición de una Objeto en Python
class Coche:
    def __init__(self, marca, modelo):
        self.marca = marca
        self.modelo = modelo

    def obtener_info(self):
        return f"Coche: {self.marca} {self.modelo}"

# Creación de una instancia (objeto) de la clase Coche
mi_coche = Coche(marca="Toyota", modelo="Corolla")

# Acceso a los atributos y llamada a un método del objeto
print(mi_coche.marca)  # Imprime: Toyota
print(mi_coche.obtener_info())  # Imprime: Coche: Toyota Corolla

File: C++/Objetos.cpp
// C++ es un lenguaje orientado a objeto como Java
// pero con una sintaxis más simple y no tiene la clase abstracta.
#include <iostream>
#include <string>
using namespace std;
// Vamos hace un objeto persona
class Persona {
    public:
    int edad;
    string identificacion;
    string nombre= "Jesus";
    string apellido= "Campos";
    };
    int main() {
        // crear un objeto de la clase persona
        Persona miPersona;
        cout << "Nombre completo: "<<miPersona.nombre<<" "<<miPersona.apellido<<endl;
        // acceder a los atributos de la clase
        miPersona.edad = 25;
        miPersona.identificacion ="10346789";
        cout << "Edad: "<<miPersona.edad<<endl;
        cout << "Identificacion: "<<miPersona.identificacion<<endl;
}